README.md

```diff
@@ -420,11 +421,103 @@ description: "This high priority class should be used for MarkLogic pods only."
+7. Repeat termination for all pods in your release and complete the upgrade process.
+
+## Upgrading MarkLogic Version in your release
+
+MarkLogic release is independent of MarkLogic Kubernetes Helm Chart release. This upgrade may be required when there is a new MarkLogic version available. Note MarkLogic is also released in Major, Minor or Patch form, pleasr refer [MarkLogic Upgrade Support](https://docs.marklogic.com/11.0/guide/release-notes/en/installation-and-upgrade.html#UUID-ae8580bb-c23d-dda7-593c-0610a633c771_section-idm203341859110072)
+
+Following is the procedure to upgrade MarkLogic version in your release:
+
+1. Update the image.repository and image.tag in the values.yaml file to new version of MarkLogic you want to upgrade to.
+```
+image:
+  repository: marklogicdb/marklogic-db
+  tag: latest-11.0.x
+```
+2. Set the upgrade strategy in values.yaml file to OnDelete. 
+```
+updateStrategy: 
+    type: OnDelete
+```
+3. Upgrade the helm chart using helm upgrade command with release name, chart name and values.yaml.
+```
+helm upgrade <release-name> <chart-name> -f <values.yaml> 
+```
+4. To start upgrade, delete the pod with smallest ordinal that is a MarkLogic bootstrap host. 
+```
+kubectl delete pod dnode-group-marklogic-0
+```
+Once the pod is terminated, new pod will be created with updated MarkLogic version and any new values updated in the values.yaml file.
+
+5. Repeat the termination for all pods in your release to complete the upgrade.
+
+6. After all pods are upgraded, access the Admin Interface on bootstrap host to check there is configuration and/or security database upgrade and/or effective version change. If yes, Admin Interface will prompt you to click OK to upgrade otherwise you are done.
+7. Verify the upgrade, check the version of MarkLogic on Admin console or by accessing server logs or you can run any required tests for your release.
+8. If you've multi group MarkLogic cluster, each release corresponding to a MarkLogic group needs to be upgraded by following above procedure.
+Note: If all the nodes in the groups are not updated to the same MarkLogic version then you will see difference in the version and effective version of the MarkLogic cluster.
+
+## Upgrading MarkLogic version and MarkLogic Kubernetes Helm Chart version at once
+
+1. If there is a requirement to upgrade MarkLogic version and MarkLogic Kubernetes Helm Chart version both at once, we recommend following upgrade Steps 1 to 5 from [Upgrading MarkLogic Version in your release](#upgrading-markLogic-version-in-your-release) and Steps 1 to 3 from [Upgrading MarkLogic Version in your release](#upgrading-markLogic-version-in-your-release).
+
+2. Once the `helm upgrade` command is executed for both, initiate terminating pods. As recommended, delete the pod-0 i.e. running MarkLogic Bootstrap host first followed by other pods. 
+```
+kubectl delete pod dnode-group-marklogic-0
```

Comment:
`kubectl delete pod <pod name>`
e.g.
`kubectl delete pod dnode-group-marklogic-0`

Reply:
update command and example